### DISCUSSION TITLE
Why does the latter fail?

### DISCUSSION THREAD

**Discussion by limaceous-bushwhacker** [lsd-rs/lsd#1047] [Q&A] [answered]
`ls -p | grep -v /` vs `lsd -F | grep -v /`

Latter fails with exit code 1.

**Comment by zwpaper** [comment, marked as answer]
`lsd -F` not indicating one item per line when redirect to grep, please try `lsd -F -1`

**Reply by limaceous-bushwhacker** [reply]
Ah, finally! This had me scratching my head. Reckon this is something that needs to be fixed?
